curriculum/challenges/english/25-front-end-development/lecture-html-fundamentals/6708382cf088b216580a9ff1.md

```diff
@@ -0,0 +1,125 @@
+---
+id: 6708382cf088b216580a9ff1
+title: What Are IDs and Classes, and When Should You Use Them?
+challengeType: 11
+videoId: nVAaxZ34khk
+dashedName: what-are-ids-and-classes
+---
+
+# --description--
+
+Watch the video and answer the questions below.
+
+# --questions--
+
+## --text--
+
+When should you use an ID versus a class?
```

Comment:
for all mentions of id's in this file, it would be better to use all lowercase and have it wrapped in backticks instead of all caps

Reply:
Also, I fell like class should be in backticks too

Reply:
✅ added